js/itk-siteimprove.js

```diff
@@ -9,17 +9,32 @@
   Drupal.behaviors.cookieMessageBehavior = {
     attach: function (context, settings) {
       var key = drupalSettings.itk_siteimprove.key;
-
+      var use_cookieinformation = drupalSettings.itk_siteimprove.use_cookieinformation;
       if (key) {
-        (function () {
-          var sz = document.createElement('script');
-          sz.type = 'text/javascript';
-          sz.async = true;
-          sz.src = '//ssl.siteimprove.com/js/siteanalyze_' + key + '.js';
-          var s = document.getElementsByTagName('script')[0];
-          s.parentNode.insertBefore(sz, s);
-        })();
+        if (use_cookieinformation == 1) {
```

Comment:
when you leave "use_cookieinformation" as a number and not a boolean, checking if the value is 1, you use your implied knowledge about the variable (that it in fact is a boolean), to check if it is set.

Reply:
Fixed